marketpulse/main/templates/fxosprice_all.html

```diff
@@ -65,7 +65,10 @@ <h3 class="title">Contributions</h3>
               {% endif %}
             </td>
             <td class="delete-contribution">
-              <a href="{{ url('main:view_fxosprice', contribution.pk) }}" class="btn btn-default btn-xs">Details</a>
+              <a href="{{ url('main:view_contribution', contribution.pk) }}"
+                 class="btn btn-default btn-xs">Details</a>
```

Comment:
I suggest to either have all the links ``edit, delete`` on the detailed view page, or show these here as well.